simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/ui/AccountInformationFragment.kt

```diff
@@ -11,33 +9,35 @@ import android.widget.Toast
 import androidx.appcompat.app.AlertDialog
 import androidx.fragment.app.Fragment
 import androidx.fragment.app.activityViewModels
+import androidx.lifecycle.ViewModelProvider
 import androidx.navigation.NavController
 import androidx.navigation.NavDirections
-import androidx.navigation.Navigation
+import androidx.navigation.fragment.findNavController
+import org.nypl.simplified.android.ktx.viewLifecycleAware
 import org.nypl.simplified.cardcreator.R
 import org.nypl.simplified.cardcreator.databinding.FragmentAccountInformationBinding
-import org.nypl.simplified.cardcreator.model.ValidateUsernameResponse
+import org.nypl.simplified.cardcreator.model.UsernameVerificationResponse
+import org.nypl.simplified.cardcreator.model.UsernameVerificationResponse.UsernameVerificationError
+import org.nypl.simplified.cardcreator.model.UsernameVerificationResponse.UsernameVerificationException
+import org.nypl.simplified.cardcreator.model.UsernameVerificationResponse.UsernameVerificationSuccess
 import org.nypl.simplified.cardcreator.utils.Cache
 import org.nypl.simplified.cardcreator.utils.hideKeyboard
-import org.nypl.simplified.cardcreator.viewmodel.UsernameViewModel
+import org.nypl.simplified.cardcreator.viewmodel.AccountInformationViewModel
 import org.slf4j.LoggerFactory
 
-class AccountInformationFragment : Fragment() {
+class AccountInformationFragment(
+  // Needed to enable isolated fragment testing with an activityViewModel
+  activityVMFactory: (() -> ViewModelProvider.Factory)? = null
+) : Fragment() {
 
   private val logger = LoggerFactory.getLogger(AccountInformationFragment::class.java)
 
-  private var _binding: FragmentAccountInformationBinding? = null
-  private val binding get() = _binding!!
+  private var binding by viewLifecycleAware<FragmentAccountInformationBinding>()
```

Comment:
Is `viewLifecycleAware` the recommendation for viewBindings?

Reply:
I don't think it's an official recommendation, just something I picked up somewhere - I like it as an easy way to deal with managing binding references in a lifecycle-aware manner